requirements/requirements.txt

```diff
@@ -27,7 +27,7 @@ statsmodels>=0.11.0
 scipy>=1.4.1
 dataclasses>=0.6; python_version < '3.7'
 # jupyterlab-plotly extension that is used to display plotly figures
-plotly>=5.5.0
+plotly>=5.8.0
```

Comment:
why?

Reply:
they also used the deprecated numpy things before 5.8